frontend/assets/css/main.css

```
@@ -1,7 +1,244 @@
+  border-bottom: none !important;
+}
+
+/* Shared utility classes used across item/search components */
+.action-btn {
+  width: 28px !important;
+  height: 28px !important;
+}
+
+.full-width {
+  width: 100%;
+  word-wrap: break-word;
```

Stylelint (17.9.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)